test/rinda/test_rinda.rb

```diff
@@ -402,7 +402,7 @@ def test_cancel_01
   end
 
   def test_cancel_02
-    skip 'this test is unstable with --jit-wait' if defined?(RubyVM::JIT) && RubyVM::JIT.enabled?
+    pend 'this test is unstable with --jit-wait' if defined?(RubyVM::JIT) && RubyVM::JIT.enabled?
```

Comment:
Can this be "omit" and still work? Are we expecting the test ever to become stable with jit-wait? 

If not, perhaps skip->omit is another way of saying skip?

(This is just me pattern-matching on this change, which was done in another standard library gem a while ago.)

Super-nice fix, with the mixin module taking care of the testing for support.

Reply:
In test-unit, `omit` is the right alternative of `skip` of minitest.
`pend` is for a case that will be fixed later.
`omit` is for a case that is never ran.

Reply:
OK, I switched `pend` to `omit`.